tests/language-service/languageservicerunner.ts

```diff
@@ -46,16 +47,34 @@ describe("language service", () => {
 })
 
 function getTestCases() {
-    const filenames: string[] = [];
+    let filenames: string[] = [];
     for (const file of fs.readdirSync(casesDir)) {
+        // ignore hidden files
         if (file[0] == ".") {
             continue;
         }
 
+        // ignore files that start with TODO_; these represent future work
+        if (file.indexOf("TODO") >= 0) {
+            continue;
+        }
+
+        // TODO support python as well
+        if (file.substr(-3) !== ".ts") {
+            continue;
```

Comment:
Emit a warning? Or we’ll forget

Reply:
I filed an issue and will put the issue number here. Again, I think work item priority should be decided outside of the code and therefore the code shouldn't force an importance by throwing a warning.

Reply:
a console.log calls does not hurt and reminds everyone about this fact. it'll save someone else an hour in the future to understand why the .ts test does not run. github issues are fragile, they get closed/buried.